CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to hg-engine
+
+Hello!  `hg-engine` is always open to contributors making pull requests for features or other battle engine facets.
+These contributions are assessed based on whether or not they are "in-scope" and then merged as they are completed.
+
+## Scope
+
+The overall scope of the project is first and foremost battle engine overhauls.  `hg-engine` aims to be a battle engine that ports all of the features from later generations to that of HeartGold.
+
+From there, we also have quality of life improvements.  These are things like repels that automatically prompt usage of another one, 
+
+## Bug Reports
+
+We use [GitHub](https://github.com/BluRosie/hg-engine/issues) issues to track bugs.
+
+The Issues list is somewhat of a TODO list that is worked through.
```

Comment:
We should probably have an Issues   Template in the future.

Reply:
what do you suggest be on there?

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

I think we can reference RHH's ones as well.

But have a general template for bugs like freezes, etc, and have a more specific template for battle engine bugs that has a section for users to post a video of the bug happening in hg-engine and another section for vanilla behaviour evidence.